rust/syntect-plugin/makefile

```diff
@@ -0,0 +1,10 @@
+PLUGIN_DIR = ~/.config/xi/plugins
```

Comment:
can we check for `XI_PLUGIN_DIR` and use that if set?

Reply:
you mean the env variable `XI_PLUGIN_DIR` or just renaming the variable to `XI_PLUGIN_DIR`?

Reply:
@eyelash I mean check for the env var. We should also do some thinking about a longer term strategy with make (a set of conventions for verifying the environment, building, and installing, that we can standardize for plugins in general) but I think this is good for now!

Reply:
@cmyr I pushed another commit but for whatever reason I can't see it here in the pull request. It works for me on Linux but I don't know what version of make macOS uses so you need to test it there as well.